contracts/hardhat.config.ts

```diff
@@ -131,6 +131,7 @@ const config: HardhatUserConfig = {
   },
   networks: {
     hardhat: {
+      chainId: 1337,
```

Comment:
Decided to enforce hardhat using the same chainId as the geth node. This means we don't have to change config when moving between the two. e.g. If you wanted to run quill against the hardhat node, you'd have to change the config file to use a different chainId.

Deemed changing hardhat chainId over changing the geth chainId mainly because it was easier. `--dev` mode for geth is [not meant to be configured](https://github.com/ethereum/go-ethereum/issues/21120#issuecomment-633646596). If we wanted to change the chainId, looks like we'd have to run a [private network with a custom genesis file](https://ethereum.stackexchange.com/questions/25359/geth-chain-option) which seems like a lot more effort than necessary.